pkg/cli/schemaherokubectlcli/get_migrations.go

```diff
@@ -95,27 +96,37 @@ func GetMigrationsCmd() *cobra.Command {
 			}
 
 			rows := [][]string{}
-			for _, migration := range matchingMigrations {
-				// TODO should we show these?
-				if migration.Status.ExecutedAt > 0 {
-					continue
-				}
-				if migration.Status.RejectedAt > 0 {
-					continue
+			for _, m := range matchingMigrations {
+				table, err := migration.TableFromMigration(context.Background(), &m)
+				if err != nil {
+					return err
 				}
-				if migration.Status.ApprovedAt > 0 {
-					continue
+
+				database, err := migration.DatabaseFromTable(context.Background(), table)
+				if err != nil {
+					return err
 				}
 
-				rows = append(rows, []string{
-					migration.Name,
-					"<Database Unknown>",
-					migration.Spec.TableName,
-					timestampToAge(migration.Status.PlannedAt),
-					timestampToAge(migration.Status.ExecutedAt),
-					timestampToAge(migration.Status.ApprovedAt),
-					timestampToAge(migration.Status.RejectedAt),
-				})
+				isIncluded := true
```

Comment:
is this intended?

Reply:
yeah, there's some filtering work that's in progress....  cli flags to filter approved at, executed at, or reject at.  in this implementation here, they are all included